### ISSUE TITLE
New Rule: no-invalid-named-grid-areas

### ISSUE THREAD

**Issue by xbinaryx** [eslint/css#158]
### Rule details

Disallow invalid named grid areas.

### What type of rule is this?

Warns about a potential problem

### Example code

```css
a { grid-template-areas: "a a a"
                         "b b b b"; }
```

### Prior Art

[named-grid-areas-no-invalid](https://stylelint.io/user-guide/rules/named-grid-areas-no-invalid/) from stylelint

### Participation

- [x] I am willing to submit a pull request to implement this rule.

### Additional comments

For a named grid area to be valid, all strings must define:

- the same number of cell tokens
- at least one cell token

And all named grid areas that spans multiple grid cells must form a single filled-in rectangle.

**Comment by nzakas** [comment]
Just to double-check, this isn't already caught by `no-invalid-properties`, correct?
